crypto/build.info

```diff
@@ -2,7 +2,7 @@
 # there for further explanations.
 SUBDIRS=objects buffer bio stack lhash rand evp asn1 pem x509 x509v3 conf \
         txt_db pkcs7 pkcs12 ui kdf store \
-        md2 md4 md5 sha mdc2 hmac ripemd whrlpool poly1305 blake2 \
+        md2 md4 md5 sha mdc2 gmac hmac ripemd whrlpool poly1305 blake2 \
```

Comment:
Why is this being added???

It should have been there already.

Reply:
@levitte mentioned it is through the merge that it came in?